deployment/ccip/changeset/test_helpers.go

```diff
@@ -1016,3 +1016,13 @@ func deployTransferTokenOneEnd(
 
 	return tokenContract.Contract, tokenPool.Contract, nil
 }
+
+func DefaultRouterMessage(receiverAddress common.Address) router.ClientEVM2AnyMessage {
```

Comment:
This doesn't seem to be used

Reply:
It's used here: TestCCIPReader_GetExpectedNextSequenceNumber